src/utils/queries/regex.mjs

```diff
@@ -0,0 +1,296 @@
+// ============================================================================
```

Comment:
Instead of doing this, why not different files? These section blocks aren't a common thing in JavaScript.